docs/feature-gates.md

```diff
@@ -413,3 +414,12 @@ bandwidth for all egress traffic belonging to an Egress. Refer to this [document
 
 This feature leverages OVS meters to do the actual rate-limiting, therefore this feature requires OVS meters
 to be supported in the datapath.
+
+### L7Visibility
+
+`L7Visibility` enables users to collect and visualize application-layer flow data in Theia using Pod or Namespace
```

Comment:
I think we should remove "visualize" and "Theia", as they are not really supported today.